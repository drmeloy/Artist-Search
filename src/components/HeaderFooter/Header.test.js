import React from 'react';
import { shallow }  from 'enzyme';
import Header from './Header';

describe('Headerz component', () => {
  it('renders a header', () => {
    const wrapper = shallow(<Header />); 
    expect(wrapper).toMatchSnapshot(); 
  });
});
