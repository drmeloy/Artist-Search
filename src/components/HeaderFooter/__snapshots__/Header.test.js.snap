// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headerz component renders a header 1`] = `
<Fragment>
  <header
    className="Header"
  >
    <h1>
      Artist Search Application
    </h1>
  </header>
</Fragment>
`;
